packages/loader/driver-utils/.eslintrc.cjs

```diff
@@ -21,6 +18,16 @@ module.exports = {
 			files: ["*.spec.ts", "src/test/**"],
 			rules: {
 				"import/no-nodejs-modules": "off", // Node libraries are OK for test files.
+				// Tests often use flexible types and anonymous functions; relax strict rules here only.
+				"@typescript-eslint/explicit-function-return-type": "off",
+				"@typescript-eslint/explicit-module-boundary-types": "off",
+				"@typescript-eslint/no-explicit-any": "off",
+				"@typescript-eslint/no-unsafe-assignment": "off",
+				"@typescript-eslint/no-unsafe-member-access": "off",
+				"@typescript-eslint/no-unsafe-argument": "off",
+				"@rushstack/no-new-null": "off",
+				"jsdoc/require-description": "off",
+				"no-empty": "off",
```

Comment:
Do we have an idea of how bad it'd be to do this in a targeted manner? I.e., a few VS several dozens of individual disables?